par2/decoder.go

```diff
@@ -518,8 +518,8 @@ func (d *Decoder) newCoderAndShards() (rsec16.Coder, [][]byte, error) {
 }
 
 // Verify checks that all file (and maybe parity) data are consistent
-// with each other, and returns the result. If any data (or maybe
-// parity) files are missing, Verify returns false.
+// with each other, and returns the result. The returned bool is true
+// if the data are uncorrupted, in all other cases false.
 func (d *Decoder) Verify(checkParity bool) (bool, error) {
```

Comment:
I think the thing to do here is to remove the `checkParity` param, and make the returned `bool` mean whether or not there's repairable damage. (May as well name the return parameter in that case.)